### PR TITLE
New-app-templates

### PR DESCRIPTION
**Pull request by dynamite-bud**
Add tutorials for creating and deploying workers on Wasmer Edge:

Adds following tutorials:
- [x] Python worker
- [x] JavaScript worker

### CODE REVIEW

File: pages/edge/quickstart/js-wintercg.mdx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+### Step 3.1: Initialize the JavaScript Service Worker Starter template
+
+```shell copy
+$ wasmer app create
+App type:
+  Static website
+  HTTP server
+  Browser shell
+> JS Worker (experimental)
+  Python Worker
```
Comment (syrusakbary) [inline pages/edge/quickstart/js-wintercg.mdx:29]: This should be:

```
  Python Webserver
```

File: pages/edge/quickstart/js-wintercg.mdx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+You directory composition should look like this:
+
+<FileTree>
+  <FileTree.Folder name="src" defaultOpen>
+    <FileTree.File name="index.js" />
+  </FileTree.Folder>
+  <FileTree.File name="wasmer.toml" />
+  <FileTree.File name="app.yaml" />
+</FileTree>
+
```
Comment (syrusakbary) [inline pages/edge/quickstart/js-wintercg.mdx:52]: We should add here the content of `index.js`, so people know what to modify

File: pages/edge/quickstart/python.mdx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+### Step 3.1: Initialize the Python Application Starter template
+
+```shell copy
+$ wasmer app create
+App type:
+  Static website
+  HTTP server
+  Browser shell
+  JS Worker (experimental)
+> Python Worker
```
Comment (syrusakbary) [inline pages/edge/quickstart/python.mdx:30]:
```suggestion
> Python Webserver
```

File: pages/edge/quickstart/python.mdx [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+You directory composition should look like this:
+
+<FileTree>
+  <FileTree.Folder name="src" defaultOpen>
+    <FileTree.File name="main.py" />
+  </FileTree.Folder>
+  <FileTree.File name="wasmer.toml" />
+  <FileTree.File name="app.yaml" />
+</FileTree>
+
```
Comment (syrusakbary) [inline pages/edge/quickstart/python.mdx:53]: We should add the contents of `main.py` here as well

File: pages/edge/quickstart/python.mdx (hunk 3 above)
Comment (dynamite-bud) [reply pages/edge/quickstart/python.mdx:30]: It's in the CLI now. Nothing can be done till the next wasmer release

File: pages/edge/quickstart/js-wintercg.mdx (hunk 1 above)
Comment (dynamite-bud) [reply pages/edge/quickstart/js-wintercg.mdx:29]: Nothing can be done till next wasmer release

**Pull request merged**